src/test/fixtures/profiles/processed-profile.js

```diff
@@ -506,29 +506,52 @@ function _buildThreadFromTextOnlyStacks(
   return thread;
 }
 
-export function getNetworkMarkers(startTime: number, id: number) {
-  const name = `Load ${id}: https://mozilla.org`;
+type NetworkMarkersOptions = {|
+  uri: string,
+  id: number,
+  startTime: number,
+  fetchStart: number,
+  endTime: number,
+  payload: $Shape<NetworkPayload>,
+|};
+
+export function getNetworkMarkers(options: $Shape<NetworkMarkersOptions> = {}) {
+  // Default values
+  options = {
```

Comment:
What about...
```js
const { uri, id, startTime, fetchStart, endTime, payload } = {
    uri: 'https://mozilla.org',
    id: 0,
    startTime: 0,
    fetchStart: (options.startTime || 0) + 0.5,
    endTime: (options.fetchStart || (options.startTime || 0) + 0.5) + 0.5,
    payload: {},
    ...options,
  };
```
My mind struggled a bit understanding the re-assigning of `options`, plus including it in the object spread.